Resources/views/Block/default.html.twig

```diff
@@ -1,3 +1,5 @@
+{% if content is not empty %}
```

Comment:
Ok fine, but it seems that i add a bug in my previous pull request refactoring, i set the default template to the atomic block renderer, it might be atomic.html.twig, i will fixup